[2025-04-02T17:26:18.795-0400] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-02T17:26:18.802-0400] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: azure_parallel_test.azure_call_1 manual__2025-04-02T21:13:05.132505+00:00 [queued]>
[2025-04-02T17:26:18.804-0400] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: azure_parallel_test.azure_call_1 manual__2025-04-02T21:13:05.132505+00:00 [queued]>
[2025-04-02T17:26:18.804-0400] {taskinstance.py:2303} INFO - Starting attempt 3 of 1
[2025-04-02T17:26:18.808-0400] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): azure_call_1> on 2025-04-02 21:13:05.132505+00:00
[2025-04-02T17:26:18.811-0400] {standard_task_runner.py:63} INFO - Started process 15293 to run task
[2025-04-02T17:26:18.814-0400] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'azure_parallel_test', 'azure_call_1', 'manual__2025-04-02T21:13:05.132505+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/azure_parallel_test_dag.py', '--cfg-path', '/var/folders/yx/9w9ypstn1fq5mxvr0l83chw40000gp/T/tmp0vkln7iw']
[2025-04-02T17:26:18.815-0400] {standard_task_runner.py:91} INFO - Job 13: Subtask azure_call_1
[2025-04-02T17:26:18.830-0400] {task_command.py:426} INFO - Running <TaskInstance: azure_parallel_test.azure_call_1 manual__2025-04-02T21:13:05.132505+00:00 [running]> on host pgadhe-mb-W2XPM
[2025-04-02T17:26:18.850-0400] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='azure_parallel_test' AIRFLOW_CTX_TASK_ID='azure_call_1' AIRFLOW_CTX_EXECUTION_DATE='2025-04-02T21:13:05.132505+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-02T21:13:05.132505+00:00'
[2025-04-02T17:26:18.850-0400] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-02T17:28:29.929-0400] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-04-02T17:28:29.936-0400] {process_utils.py:132} INFO - Sending Signals.SIGTERM to group 15293. PIDs of all processes in the group: [15293]
[2025-04-02T17:28:29.936-0400] {process_utils.py:87} INFO - Sending the signal Signals.SIGTERM to group 15293
[2025-04-02T17:28:33.341-0400] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-02T17:28:33.347-0400] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: azure_parallel_test.azure_call_1 manual__2025-04-02T21:13:05.132505+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2025-04-02T17:28:33.349-0400] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: azure_parallel_test.azure_call_1 manual__2025-04-02T21:13:05.132505+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2025-04-02T17:28:33.349-0400] {local_task_job_runner.py:163} INFO - Task is not able to be run
